navigation/navigation-runtime-ktx/src/main/java/androidx/navigation/NavController.kt

```diff
@@ -26,3 +31,19 @@ public inline fun NavController.createGraph(
     @IdRes startDestination: Int,
     builder: NavGraphBuilder.() -> Unit
 ): NavGraph = navigatorProvider.navigation(id, startDestination, builder)
+
+/**
+ * Creates and returns a [Flow] that will emit the currently active [NavBackStackEntry] whenever
+ * it changes. If there is no active [NavBackStackEntry], no item will be emitted.
+ */
+@Suppress("EXPERIMENTAL_API_USAGE")
```

Comment:
I think it should be marked as experimental api. We won't be able to release it as stable until callbackFlow will be stable (androidx gives binary compatibility guarantees that could be broken by callbackFlow)

Reply:
Just to make sure I understand correctly - since it is part of the public API `@ExperimentalCoroutinesApi` is required. If it were an internal function, `@Suppress` would be fine (as it does not cause binary incompatible changes), correct?